source/common/common/logger.h

```diff
@@ -50,6 +50,7 @@ const static bool should_log = true;
   FUNCTION(decompression)                                                                          \
   FUNCTION(dns)                                                                                    \
   FUNCTION(dubbo)                                                                                  \
+  FUNCTION(dynamic_modules)                                                                        \
```

Comment:
Could you add this at the end of the list? This change all these enum's value which may bring potential risk.

Reply:
https://github.com/envoyproxy/envoy/pull/37296/commits/34af90e8a0216a32b9ff26335420b081070b86e6 done